docs/lib/analytics/fragments/android/getting-started.md

```diff
@@ -1,4 +1,4 @@
-When your backend is successfully updated, there should be two newly created files: `amplifyconfiguration.json` and `awsconfiguration.json` in your project folder.
+When your backend is successfully updated, there should be one newly created file: `amplifyconfiguration.json` in your project folder.
 
 ## Install Amplify libraries and tools
```

Comment:
Let's point them to the project setup guide instead and remove this project build.gradle stuff.

Reply:
Change all versions from 0.10.0 to 1.0.0

Reply:
Is this content duplicated below? What is this file?

Reply:
Oh, /android/getting-started.md is originally here, it was restructured to getting-started/* and native_common/getting-started/common.md. I should delete these two getting-started.md

Reply:
Deleted /android/getting-started.md and /ios/getting-started.md